__tests__/api/__utils__/seeds/users_and_follow_seeds.js

```diff
@@ -102,7 +108,8 @@ const generateUserAndFollowSeeds = async () => {
   return {
     users: bulks,
     followers,
-    followings
+    followings,
+    anonymousUsername
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Returning `anonymousUsername` from the `generateUserAndFollowSeeds` function could be useful for tests that need to know the generated anonymous username. Ensure that this returned value is used appropriately in the tests.
```diff
+ anonymousUsername
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->